usersession/agent/rest_api.go

```diff
@@ -245,26 +245,28 @@ func postPendingRefreshNotification(c *Command, r *http.Request) Response {
 	}
 
 	// TODO: this message needs to be crafted better as it's the only thing guaranteed to be delivered.
-	summary := fmt.Sprintf(i18n.G("Pending update of %q snap"), refreshInfo.InstanceName)
+	summary := fmt.Sprintf(i18n.G("Update available for %s."), refreshInfo.InstanceName)
 	var urgencyLevel notification.Urgency
 	var body, icon string
 	var hints []notification.Hint
 
-	plzClose := i18n.G("Close the app to update now")
 	if daysLeft := int(refreshInfo.TimeRemaining.Truncate(time.Hour).Hours() / 24); daysLeft > 0 {
 		urgencyLevel = notification.LowUrgency
-		body = fmt.Sprintf("%s (%s)", plzClose, fmt.Sprintf(
-			i18n.NG("%d day left", "%d days left", daysLeft), daysLeft))
+		body = fmt.Sprintf(
+			i18n.NG("Close the application to update now. It will update automatically in %d day.",
```

Comment:
I wonder whether s/automaticlly/forcefully/ would make it a bit clearer that not updating while running is often preferable?

Reply:
@robert-ancell I will merge it as is but it would be nice if you could bring this feedback back to the designers to ensure that it's considered. We can always do a followup if it is.